tools/ds-identify

```diff
@@ -1565,6 +1565,29 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
+dscheck_WSL() {
+    if [ ! "${DI_UNAME_KERNEL_NAME}" = "Linux" ]; then
```

Comment:
I'm pretty sure all distros we support run ds-identify from POSIX-compliant implementations. But, there is an off chance that a minimal image may have a busybox implementation somewhere that doesn't adhere strictly to POSIX order of operators here.
In this case and below, to be clear in all cases, let's just use the `!=` operator to be safe.
```suggestion
    if [ "${DI_UNAME_KERNEL_NAME}" != "Linux" ]; then
```